### PR TITLE
fix: Show optionBar in meeting Screen

### PR DESCRIPTION
**Pull request by harinath01**

### CODE REVIEW

File: course/src/main/java/in/testpress/course/ui/MeetingOptionBarFragment.kt [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+            binding.hand.id -> {
+                callback!!.onClickHand()
+            }
+            binding.exit.id -> {
+                activity?.onBackPressed()
+            }
+        }
+    }
+
+    fun refreshHandIcon(active: Boolean){
```
Comment (Karthik-0) [inline course/src/main/java/in/testpress/course/ui/MeetingOptionBarFragment.kt:59]:
```suggestion
    fun changeHandIconColor(active: Boolean){
```

File: course/src/main/java/in/testpress/course/ui/MeetingOptionBarFragment.kt [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+            )
+        }
+        else {
+            binding.hand.backgroundTintList = ColorStateList.valueOf(
+                Color.parseColor("#aaaaaa")
+            )
+        }
+    }
+
+    fun refreshSpeakerIcon(active: Boolean) {
```
Comment (Karthik-0) [inline course/src/main/java/in/testpress/course/ui/MeetingOptionBarFragment.kt:72]:
```suggestion
    fun changeSpeakerIconColor(active: Boolean) {
```

File: course/src/main/java/in/testpress/course/ui/MeetingOptionBarFragment.kt [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+                ContextCompat.getColor(context!!, R.color.testpress_color_primary)
+            )
+        } else {
+            binding.speaker.backgroundTintList = ColorStateList.valueOf(
+                Color.parseColor("#aaaaaa")
+            )
+        }
+    }
+
+    fun refreshChatIcon(active: Boolean){
```
Comment (Karthik-0) [inline course/src/main/java/in/testpress/course/ui/MeetingOptionBarFragment.kt:84]: .

File: course/src/main/java/in/testpress/course/ui/MeetingScreenFragment.kt [hunk 4]
```diff
@@ -35,9 +45,53 @@ class MeetingScreenFragment : Fragment(), MeetingShareCallback.ShareEvent {
...
+    private fun setCallbackOnOptionBar(){
+        optionBarFragment?.setCallback(object: MeetingOptionBarFragment.Companion.MeetingOptionBarCallback {
+            override fun onClickChats() {
+                val sidebar = meetingScreenBinding.sidebar
+                if (sidebar.isVisible){
+                    meetingScreenBinding.sidebar.visibility = View.GONE
+                }else{
+                    meetingScreenBinding.sidebar.visibility = View.VISIBLE
+                }
+                renderVideo()
```
Comment (Karthik-0) [inline course/src/main/java/in/testpress/course/ui/MeetingScreenFragment.kt:63]: Why is this here?

File: course/src/main/java/in/testpress/course/ui/MeetingScreenFragment.kt [hunk 5]
```diff
@@ -35,9 +45,53 @@ class MeetingScreenFragment : Fragment(), MeetingShareCallback.ShareEvent {
...
         primaryVideoViewManager = meetingScreenBinding.primaryMeetingView.videoViewManager
         webCamVideoViewManager = meetingScreenBinding.webCamView.videoViewManager
+        audioController = inMeetingService.inMeetingAudioController
+        optionBarFragment = meetingScreenBinding.optionBar.getFragment<MeetingOptionBarFragment>()
+        setCallbackOnOptionBar()
         renderVideo()
     }
 
+    private fun setCallbackOnOptionBar(){
+        optionBarFragment?.setCallback(object: MeetingOptionBarFragment.Companion.MeetingOptionBarCallback {
```
Comment (Karthik-0) [inline course/src/main/java/in/testpress/course/ui/MeetingScreenFragment.kt:55]: This can be inherited in class and
optionBarFragment?.setCallback(this)

File: course/src/main/java/in/testpress/course/ui/MeetingScreenFragment.kt (hunk 4 above)
Comment (harinath01) [reply course/src/main/java/in/testpress/course/ui/MeetingScreenFragment.kt:63]: To rerender the video with the allocated size

**Review by Karthik-0 — APPROVED** [review]
